lib/elephant_in_the_room/sites/tag.ex

```diff
@@ -0,0 +1,23 @@
+defmodule ElephantInTheRoom.Sites.Tag do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias ElephantInTheRoom.Sites.{Site, Tag, Post}
+
+  schema "tags" do
+    field(:name, :string)
+
+    belongs_to(:site, Site, foreign_key: :site_id)
+
+    many_to_many(:posts, Post, join_through: "post_tags", on_delete: :delete_all)
+
+    timestamps()
```

Comment:
do we really need `timestamps`?

Reply:
They are added to every generated model, i don't know if we need them